features/user_sessions.feature

```diff
@@ -46,3 +46,66 @@ Feature: User sessions
     When I sign out
     And I go to the usagov's Dashboard page
     Then I should see "Security Notification"
+  
+  @javascript
+  Scenario: User sees the header with USWDS banner
+    When I go to the login page
+    Then I should see "An official website of the United States government"
+    When I press "Here’s how you know"
+    Then I should see "Official websites use .gov"
+    And I should see "A .gov website belongs to an official government organization in the United States."
+    And I should see "Secure .gov websites use HTTPS"
+    And I should see "A lock"
+    And I should see "https:// means you’ve safely connected to the .gov website. Share sensitive information only on official, secure websites."
```

Comment:
I would simplify this and the other scenarios by checking just a single string in the relevant partial. That keeps the features from being bloated and becoming unreadable. No need to check multiple elements in each partial, unless there's a specific need.

Reply:
Sure. In the footer, I am checking for the all links and their text as well. Do we need that? or I can just check couple of links?

Reply:
Yeah, I would check a minimal amount of content just to ensure that each partial appears. See comment below.